### DISCUSSION TITLE
Post Process UV cure

### DISCUSSION THREAD

**Discussion by DocRuzzy** [supermerill/SuperSlicer#1298] [Community contribution]
I would have use for a post-process UV cure similar to the milling post-processing option.  This for using UV curable resins extruded via a syringe which beyond my specific interest is highly applicable to bioprinting.

The basic process is:

After every layer switch the UV cure head and pass back and forth (like a 100% rectilinear fill) over an area defined by the skirt.
At a minimum, the UV cure head will need an offset and a spot diameter defined.
The distance between lines will be related to the spot diameter and the travel speed is crucial to the curing.

More complicated options include controlling overlap and intensity.
This is the hardware I'm using and related code.
http://hyrel3d.net/wiki/index.php/UV_and_Clench#Advanced_-_Between_Layers_2

**Comment by supermerill** [comment]
You should create an issue instead of a discussion.

**Comment by DocRuzzy** [comment]
OK, I'm respectfully reluctant to label something an issue. I already appreciate the work you do on this fork and wanted to gauge if there was any other interest.
